downloads/swan-lake-release-notes/swan-lake-2201.10.0/RELEASE_NOTE.md

```diff
@@ -81,6 +87,16 @@ To view bug fixes, see the GitHub milestone for Swan Lake Update 10 (2201.10.0)
 
 ### Improvements
 
+- Now, a Java dependency can be marked as GraalVM compatible in the Ballerina.toml as follows.
+
+``` toml
+[[platform.java11.dependency]]
+groupId = "<group-id>"
+artifactId = "<artifact-id>"
+version = "<version>"
+graalvmCompatible = true
+```
+
```

Comment:
* Introduced an experimental build option to enable memory-efficient compilation for large packages to prevent out-of-memory issues that can happen during the initial compilation which happens with a clean central cache. 

```bash
$ bal build --optimize-dependency-compilation
```